content-serve-auth.md

```diff
@@ -0,0 +1,106 @@
+# Content Server Authorization Spec
+
+## Editors
+
+- [Felipe Forbeck](https://github.com/fforbeck), [Storacha Network](https://storacha.network/)
+
+## Authors
+
+- [Felipe Forbeck](https://github.com/fforbeck), [Storacha Network](https://storacha.network/)
+
+## Abstract
+
+Content Server Authorization ensures that access to content is governed by UCAN delegations (User Controlled Authorization Network) and served by explicitly authorized services.
+This mechanism allows content owners to delegate retrieval capabilities to specific services, ensuring that only authorized entities can access the content.
+
+## Terminology
+
+- **Space**: A logical container for data, identified by a DID (Decentralized Identifier). Authorization is granted for an entire space, and all data within a space shares the same permissions.
+- **Delegation**: The act of granting specific capabilities to another entity via UCAN, and the signed document proving the delegation (also called a "**proof**").
+- **Gateway**: A service (e.g., Freeway) that facilitates content retrieval and enforces authorization policies.
+- **Delegations KV Store**: A key-value store used by the gateway to manage and validate delegations.
+
+## Delegation Flow
+
+The delegation process involves the following steps
+
+1. **Space Creation and Delegation**
+
+   - A user creates a space using the Storacha client.
+   - During space creation, the Storacha Client delegates the `space/content/serve` capability to the Storacha Gateway (e.g., Freeway) by sending a `POST /access/delegate` request containing the UCAN delegation.
```

Comment:
> to the Storacha Gateway (e.g., Freeway)

To services designed to serve content from the Storacha Network to clients. For example an IPFS Gateway.